### DISCUSSION TITLE
Import songs sorted with stable's "Date Added" sort

### DISCUSSION THREAD

**Discussion by sournub** [ppy/osu#13713] [Ideas] [answered]
I understand that how osu!stable sorts with "Date Added" is based upon the last modified time on the file in the folder, and currently osu!lazer imports however it wants with the import process and thats the "date" added, I just wanted to know if it were possible to either have it modify its last modified time on the files, or to sort it atleast in order of how it was added, but to properly sort with date added like in stable

**Comment by peppy** [comment, marked as answer]
Sounds like something we'd definitely want to support.

I've opened a tracking issue https://github.com/ppy/osu/issues/13717.
